Tests/scripts/validate_premium_packs.py

```diff
@@ -0,0 +1,250 @@
+    response = ast.literal_eval(response_data)
+    if status_code == 200:
+        logging.info("Got premium packs from server.")
+        return response["packs"], response["total"]
+
+    message = response.get('message', '')
+    logging.error(f"Failed to retrieve premium packs - with status code {status_code}\n{message}\n")
+    return None, 0
+
+
+def get_premium_packs(client: demisto_client, request_timeout: int = 999999) -> dict:
+    """Get premium packs from client.
+
+    Handle the pagination.
+
+    Args:
+        client: The demisto client to the preform request on.
+        request_timeout: Timeout of each API request
+
+    Returns:
+        Dict of premium packs as found in the server.
+        Return None if no premium packs were found.
+    """
+    server_packs, total = get_paid_packs_page(client=client,
+                                              page=0,
+                                              size=DEFAULT_PAGE_SIZE,
+                                              request_timeout=request_timeout)
+    if total <= DEFAULT_PAGE_SIZE:
+        return server_packs
+    if total % DEFAULT_PAGE_SIZE == 0:
+        pages_until_all = int(total / DEFAULT_PAGE_SIZE)
+    else:
+        pages_until_all = int(total / DEFAULT_PAGE_SIZE) + 1
+
+    for page in range(1, pages_until_all):
+        next_server_packs, _ = get_paid_packs_page(client=client,
+                                                   page=page,
+                                                   size=DEFAULT_PAGE_SIZE,
+                                                   request_timeout=request_timeout)
+        server_packs.update(next_server_packs)
+    return server_packs
+
+
+def verify_pack_in_list(pack: dict, pack_list: list, pack_list_name: str = "pack list") -> bool:
+    """Verify pack is in the pack list with same id and price.
+
+    Args:
+        pack: (Dict) pack containing an id and a price.
+        pack_list: (List[Dict]) list of packs containing id and a price.
+        pack_list_name: Name of the list for better logs.
+
+    Returns: True if pack is in list, False otherwise.
+    """
+    for pack_from_list in pack_list:
+        if pack["id"] == pack_from_list["id"]:
+            price_matches = log_message_if_statement(statement=(pack["price"] == pack_from_list["price"]),
+                                                     error_message=f'Price in pack {pack["id"]} does not match the'
+                                                                   f' price in the list. '
+                                                                   f'{pack["price"]} != {pack_from_list["price"]}',
+                                                     success_message=f'Pack {pack["id"]} is valid.')
+            return price_matches
+
+    logging.error(f'Pack {pack["id"]} is not in {pack_list_name}')
+    return False
+
+
+def verify_packs_inside(inner_packs: list, outer_packs: list, inner_packs_name: str, outer_packs_name: str) -> bool:
```

Comment:
The function name is not so indicative, I'd use a different one.

Reply:
verify_packs_are_contained ?
verify_pack_list_inside_other_pack_list ?
verify_pack_list_contains_other_pack_list ?
What will you prefer?